proton-j/src/main/java/org/apache/qpid/proton/engine/impl/EventImpl.java

```diff
@@ -68,12 +69,19 @@ void clear()
     }
 
     @Override
-    public Type getType()
+    public EventType getEventType()
     {
         return type;
     }
 
     @Override
+    public Type getType() {
+        if (type instanceof Type)
```

Comment:
Yes, 'drop from Type' as in move somewhere else, not get rid of entirely as they are obviously used.